packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-option-multi/gux-option-multi.tsx

```diff
@@ -118,6 +124,35 @@ export class GuxOptionMulti {
     }
   }
 
+  private renderText(): JSX.Element {
+    // The gux-slot-container attribute is used in gux-listbox-multi and gux-dropdown-multi as a selector to get the slotted gux-option-multi text.
+    // This attribute is required because we need to get the slotted text and exclude the screen reader text.
+    if (this.hasSubtext) {
+      return (
+        <div class="gux-option-text">
+          <gux-truncate
+            gux-slot-container
+            ref={el => (this.truncateElement = el)}
+          >
+            <slot />
+          </gux-truncate>
+          <div class="gux-subtext">
+            <slot name="subtext"></slot>
+          </div>
+        </div>
+      ) as JSX.Element;
+    } else {
+      return (
+        <gux-truncate
+          gux-slot-container
+          ref={el => (this.truncateElement = el)}
+        >
+          <slot />
+        </gux-truncate>
+      ) as JSX.Element;
+    }
+  }
+
```

Comment:
Could this simplified? Perhaps to something like
```
 <div class="gux-option-text">
     <gux-truncate
            gux-slot-container
            ref={el => (this.truncateElement = el)}
     >
            <slot />
     </gux-truncate>
     <slot  class="gux-subtext" name="subtext"></slot>
 </div>
```

Also instead of the class on the slot, you can also use ::slotted in the css. Either way possibly removes the need for the if-statement for hasSubtext.
The div wrapper might be able to stay for both cases or possibly removed entirely and the styles be put on the host `gux-show-subtext` class depending on how it affects things with the renderCustomOptionInstructions (since its position is absolute, it _should_ be kicked out of the flexbox flow)

Example in the renderText:
```
[
     <gux-truncate
            gux-slot-container
            ref={el => (this.truncateElement = el)}
     >
            <slot />
     </gux-truncate>,
     <slot  class="gux-subtext" name="subtext"></slot>
] as JSX.Element;
```

Reply:
Done

Reply:
You're on a Macbook, right? There is a built-in screenreader that, if it works there, you can be sure it will work elsewhere, but if it doesn't it could still work on other more common third-party screenreaders